feat: v5 (#774)
vercel/styled-jsx: .github/ISSUE_TEMPLATE.md

⚠️ IMPORTANT ⚠️

If you need help or have a question about styled-jsx please ask it on Spectrum https://spectrum.chat/styled-jsx or join our Slack community at https://vercel.chat and ask it in the `#next` channel.

Before you open a new issue please take a look at our [**Frequent Asked Questions**](https://github.com/vercel/styled-jsx#faq) and [**open issues**](https://github.com/vercel/styled-jsx/issues).

Remember, often time asking in chat or looking at the FAQ/issues can be \*faster than waiting for us to reply to a new issue\*\*.

If you are here to report a bug or request a feature please remove this introductory section and fill out the information below!

---

#### Do you want to request a _feature_ or report a _bug_?

#### What is the current behavior?

#### If the current behavior is a bug, please provide the steps to reproduce and possibly a minimal demo or testcase in the form of a Next.js app, CodeSandbox URL or similar

#### What is the expected behavior?

#### Environment (include versions)

> - Version of styled-jsx (or next.js if you're using with it):

Reviewer: ```suggestion
- Version of styled-jsx (or next.js if it's being used):
```
